academy/ibc-dev/ibcapp_packet.md

```diff
@@ -93,36 +93,40 @@ message MsgSendIbcTopRankResponse {
 ```
 <HighlightBox type="info">
 
-The proto message `MsgSendIbcTopRank` includes the field `timeoutTimestamp` which is added by Ignite CLI when scaffolding an IBC packet. It is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away from the user to have to add this manually.
+The proto message `MsgSendIbcTopRank` includes the field `timeoutTimestamp`, which is added by Ignite CLI when scaffolding an IBC packet. This is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away, making it necessary for the user to add this manually.
 
 </HighlightBox>
 
-**Note**: the proto definitions will be compiled into `types/packet.pb.go` and `types/tx.pb.go`.
+<HighlightBox type="note">
+
+The proto definitions will be compiled into `types/packet.pb.go` and `types/tx.pb.go`.
+
+</HighlightBox>
 
 ## CLI commands
 
 Ignite CLI also creates CLI commands to send packets and adds them to the `client/cli/` folder.
 
-We can thus send packets from the CLI with the following command:
+Packets can be sent from the CLI with the following command:
 ```bash
-$ leaderboardd tx leaderboard send-ibc-top-rank [portID] [channelID] [playerId] [rank] [score]
+$ leaderboard tx leaderboard send-ibc-top-rank [portID] [channelID] [playerId] [rank] [score]
```

Comment:
leaderboardd is correct: `binary` = `chain-name` + 'd' :)

Reply:
This is what happens when I overstep my bounds and venture into the code zone. I'll stay in my lane next time!

Reply:
No worries, impressed by the ability to edit language in a piece this long and technical ;)